pydevd_attach_to_process/windows/attach.cpp

```diff
@@ -108,7 +107,7 @@ struct InitializeThreadingInfo {
     PyImport_ImportModule* pyImportMod;
     PyEval_Lock* initThreads;
 
-    std::mutex mutex;
+    CRITICAL_SECTION cs;
```

Comment:
This fixes a problem with attach in 3.12. The mutex was throwing an access violation. This worked more consistently.